### PR TITLE
feat: #130 Create new DayReport

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

**Review by Howl404 — APPROVED** [review]

File: mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts [hunk 1]
```diff
@@ -30,6 +30,21 @@ export class CurrentProblemDAL {
     return currentProblem;
   }
 
+  /**
+   * Create new CurrentProblem
+   * @return {string} Uuid of new CurrentProblem
+   */
+  public static async createNewCurrentProblem(): Promise<string> {
+    const currentProblemWithoutUuid: NewCurrentProblemDTO = {
+      description: "empty current problem",
```
Comment (tritonJS826) [inline mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts:39]: Weird default value

File: mw-webapp/src/dataAccessLogic/JobDoneDAL.ts [hunk 2]
```diff
@@ -28,6 +29,22 @@ export class JobDoneDAL {
     return jobDone;
   }
 
+  /**
+   * Create new JobDone
+   * @return {string} Uuid of new JobDone
+   */
+  public static async createNewJobDone(): Promise<string> {
+    const jobDoneWithoutUuid: NewJobDoneDTO = {
+      description: "empty job done",
```
Comment (tritonJS826) [inline mw-webapp/src/dataAccessLogic/JobDoneDAL.ts:38]: Weird default value, check everywhere

File: mw-webapp/src/service/PlanForNextPeriodService.ts
Comment (tritonJS826) [inline mw-webapp/src/service/PlanForNextPeriodService.ts:1]: I don't understand what does mean word 'new' in your comments and code,please check

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/service/PlanForNextPeriodService.ts
Comment (Ekaterina1994) [reply mw-webapp/src/service/PlanForNextPeriodService.ts:1]: Deleted new from comments and renamed types

File: mw-webapp/src/dataAccessLogic/JobDoneDAL.ts (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/dataAccessLogic/JobDoneDAL.ts]: Changed

File: mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts]: Changed

**Review by SergioAJS — APPROVED** [review]

**Pull request merged**